app/views/posts/index.html.erb

```diff
@@ -1 +1,47 @@
-<h1>LISTS OF THE POSTS TO THE SPECIFIC USER</h1>
\ No newline at end of file
+<ul class="users_container">
+  <li>
+    <div class="user_img">
+      <img src="<%= @user.Photo%>" alt="Photo">
+    </div>
+    <div class="user_details">
+      <h2><%= @user.Name %></h2>
+      <p>Number of Posts: <%= @user.Posts_counter.nil? ? '0' : @user.Posts_counter %></p>
+    </div>
+  </li>
+  <% @posts.each_with_index do |post, index| %>
+    <%= link_to user_post_path(@user.id, post.id) do %>
+      <li class="user_all_posts">
+        <div class="user_bio">
+          <h3>Post#<%= index+1 %></h3>
+          <p><%= post.Text.size > 100 ? "#{post.Text[0..100]} ..." : post.Text %></p>
+          <div class="user_recent_posts">
+            <span>
+              Comments: <%= post.comments.count %>,
+            </span>
+            <span>
+              Likes: <%= post.Likes_counter.nil? ? '0' : post.Likes_counter %>
+            </span>
+          </div>
+        </div>
+
+        <div class="user_post_comments">
+          <% if post.comments.empty? %>
+            <div class="user_post_comment">
+              <p>No comments</p>
+            </div>
+          <% else %>
+            <% post.comments.each_with_index do |comment, index| %>
+              <div class="user_post_comment">
+                <h3><%= index + 1 %>. <%= comment.user.Name %></h3>
+                <p><%= comment.Text %></p>
```

Comment:
- [ ] Its a requirement that here we use `five_recent_comments` created in the post model to render 5 most recent comments in the `app/views/posts/index.html.erb` page. Kindly do so.